src/Microsoft.NetCore.Analyzers/Core/Security/DoNotUseDeprecatedSecurityProtocols.cs

```diff
@@ -54,54 +55,32 @@ public override void Initialize(AnalysisContext context)
             context.RegisterCompilationStartAction(
                 (CompilationStartAnalysisContext compilationStartAnalysisContext) =>
                 {
-                    var securityProtocolTypeTypeSymbol = compilationStartAnalysisContext.Compilation.GetTypeByMetadataName(WellKnownTypeNames.SystemNetSecurityProtocolType);
```

Comment:
I think the plan was to either use a type provider for this or to wrap the call to Compilation.GetTypeByMetadataName in a method in WellKnownTypes. Should we do one of those?

Reply:
That's what I'm doing, using WellKnownTypeProvider.  😁